app/core/Tracker.coffee

```diff
@@ -59,12 +57,6 @@ module.exports = class Tracker extends CocoClass
     @trackEventInternal('Identify', {id: me.id, traits})
     return unless @shouldTrackExternalEvents()
 
-    if me.isTeacher(true) and @segmentLoaded and not me.get('unsubscribedFromMarketingEmails')
```

Comment:
` me.get('unsubscribedFromMarketingEmails')` ignoring this because:

1. It doesn't make sense to me
2. Subscriptions will be handled via customer.io work